pyproject.toml

```
@@ -153,3 +153,14 @@ tag_format = "{version}"
 major_types = ["breaking"]
 minor_types = ["feat"]
 patch_types = ["fix", "perf"]
+
+
+[tool.pytest.ini_options]
+addopts = """
+            --reruns 3 
+            --only-rerun requests.exceptions.ReadTimeout 
+            --only-rerun huggingface_hub.errors.HfHubHTTPError 
+            --only-rerun huggingface_hub.errors.LocalEntryNotFoundError
+            --only-rerun FileNotFoundError
+            --durations=5
```

It would be nice to have some documentation on these. Could we add a comment. E.g. not sure why FileNotFoundError is there?

E.g. thinking:
```suggestion
            --reruns 3 
            --only-rerun requests.exceptions.ReadTimeout 
            --only-rerun huggingface_hub.errors.HfHubHTTPError # HF is unavailable, e.g. seen here: https://github.com/embeddings-benchmark/mteb/actions/runs/13275350693/job/37093688544
            --only-rerun huggingface_hub.errors.LocalEntryNotFoundError
            --only-rerun FileNotFoundError
            --durations=5
```

---

https://github.com/embeddings-benchmark/mteb/actions/runs/13302915091/job/37147507251?pr=2029
`FileNotFoundError` it is also a error from HF Lib.

---

perfect, great to have these when we go over the CI again next time